### ISSUE TITLE
Reserved Transactions always empty in new Mode

### ISSUE THREAD

**Issue by daniel-mller-87** [grindsa/dkb-robo#52]
hello there

when i try to obtain open transactions I always end up with an empty dict. Is the keyword reserved correct? I'm using the new method.

```
tlist = dkb.get_transactions("link", "account", "01.11.2023", "14.11.2025", "reserved" )
 pprint(tlist)
```

**Comment by daniel-mller-87** [comment]
I got the solution. It's no longer called "reserved" the name is "pending" with the new architecture

**Comment by grindsa** [comment]
Thank you for spotting this and for finding a solution yourselves. I will implement a small change and allow the "reserved" keyword for backward compatibility reason and update the documentation accordingly.

**Comment by daniel-mller-87** [comment]
You're welcome. Happy to help.

**Comment by grindsa** [comment]
I added a fix to the [devel](https://github.com/grindsa/dkb-robo/tree/devel) branch which should address your issue. Feel free to test. Official release will follow within the next few days...

**Comment by grindsa** [comment]
Fix got included in [v0.25](https://github.com/grindsa/dkb-robo/releases/tag/0.25). Thus I am closing the issue...
